src/app/component/auth/components/sign-up/sign-up.component.ts

```diff
@@ -52,13 +50,14 @@ export class SignUpComponent implements OnInit {
     this.passwordConfirmErrorMessageBackEnd = null;
   }
 
-  private onSubmit(userOwnRegister: UserOwnSignUp): void {
+  public onSubmit(userOwnRegister: UserOwnSignUp): void {
     this.setNullAllMessage();
     this.loadingAnim = true;
     this.userOwnSecurityService.signUp(userOwnRegister)
-      .subscribe({
-        next:  this.onSubmitSuccess.bind(this),
-        error: this.onSubmitError.bind(this)
+      .subscribe((data) => {
+        this.onSubmitSuccess(data);
+      }, (error) => {
+        this.onSubmitError(error);
```

Comment:
```js
.subscribe((data) => this.onSubmitSuccess(data), (error) => this.onSubmitError(error))
```
If you have only one short line in a body of an arrow function - you shouldn't write it from new line. A single-lined arrow function looks better. Excepting `no-confusing-arrow`

Reply:
> ```js
> .subscribe((data) => this.onSubmitSuccess(data), (error) => this.onSubmitError(error))
> ```
> 
> If you have only one short line in a body of an arrow function - you shouldn't write it from new line. A single-lined arrow function looks better. Excepting `no-confusing-arrow`
> 
> Change here and anywhere.

Fixed